### PR TITLE
Graph Team Tag CRUD - NodeJs

### PR DESCRIPTION
**Pull request by HunaidHanfee-MSFT**

### CODE REVIEW

File: samples/graph-teams-tag/nodejs/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "02-08-2022 00:02:15"
+---
+
+# This is a sample application shows the usage of Graph CRUD operations related to team tags.
```
Comment (ChetanSharma-msft) [inline samples/graph-teams-tag/nodejs/README.md:15]: This is a sample application that shows the usage of Graph CRUD operations related to team tags.

File: samples/graph-teams-tag/nodejs/client/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (ChetanSharma-msft) [inline samples/graph-teams-tag/nodejs/client/README.md:1]: We already have a file outside in nodejs folder then why we need this ?

File: samples/graph-teams-tag/nodejs/client/public/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Meeting Stage View Sample"
+    />
+    <title>Meetings Stage View</title>
```
Comment (ChetanSharma-msft) [inline samples/graph-teams-tag/nodejs/client/public/index.html:11]: This is not meeting stage view sample :)

File: samples/graph-teams-tag/nodejs/client/src/components/view-or-edit-tag.jsx [hunk 4]
```diff
@@ -0,0 +1,282 @@
+// <copyright file="view-or-edittag.tsx" company="Microsoft Corporation">
```
Comment (ChetanSharma-msft) [inline samples/graph-teams-tag/nodejs/client/src/components/view-or-edit-tag.jsx:1]: File name should be like view-edit-tag.jsx

File: samples/graph-teams-tag/nodejs/teamsAppManifest/manifest.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    "icons": {
+      "color": "color.png",
+      "outline": "outline.png"
+    },
+    "name": {
+      "short": "Manage Team Tags",
+      "full": "Manage Team Tags"
+    },
+    "description": {
+      "short": "This example can be used to manage Teams team tags.",
```
Comment (ChetanSharma-msft) [inline samples/graph-teams-tag/nodejs/teamsAppManifest/manifest.json:22]: This example is used to manage Teams team tags.

File: samples/graph-teams-tag/nodejs/teamsAppManifest/manifest.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+      "full": "This example can be used to manage Teams team tags. You can create, update, add or remove members. All of Graph CRUD operations related to tags can be performed within this sample."
```
Comment (ChetanSharma-msft) [inline samples/graph-teams-tag/nodejs/teamsAppManifest/manifest.json:23]: This example is used to manage Teams team tags.

File: samples/graph-teams-tag/nodejs/README.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates how to use CRUD Graph operations related to team tags.
+products:
```
Comment (ChetanSharma-msft) [inline samples/graph-teams-tag/nodejs/README.md:4]: Pleaser verify if all the images are visible properly in Readme file.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Reviewed and added my comments.

File: samples/graph-teams-tag/nodejs/client/README.md (hunk 2 above)
Comment (HunaidHanfee-MSFT) [reply samples/graph-teams-tag/nodejs/client/README.md:1]: No, removed

File: samples/graph-teams-tag/nodejs/client/public/index.html (hunk 3 above)
Comment (HunaidHanfee-MSFT) [reply samples/graph-teams-tag/nodejs/client/public/index.html:11]: Done 😄

File: samples/graph-teams-tag/nodejs/client/src/components/view-or-edit-tag.jsx (hunk 4 above)
Comment (HunaidHanfee-MSFT) [reply samples/graph-teams-tag/nodejs/client/src/components/view-or-edit-tag.jsx:1]: Done

File: samples/graph-teams-tag/nodejs/teamsAppManifest/manifest.json (hunk 5 above)
Comment (HunaidHanfee-MSFT) [reply samples/graph-teams-tag/nodejs/teamsAppManifest/manifest.json:22]: Done

File: samples/graph-teams-tag/nodejs/teamsAppManifest/manifest.json (hunk 6 above)
Comment (HunaidHanfee-MSFT) [reply samples/graph-teams-tag/nodejs/teamsAppManifest/manifest.json:23]: Done

File: samples/graph-teams-tag/nodejs/README.md (hunk 7 above)
Comment (HunaidHanfee-MSFT) [reply samples/graph-teams-tag/nodejs/README.md:4]: Yes they are visible

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**